spec/features/dashboard_spec.rb

```diff
@@ -21,7 +21,7 @@
       sign_in_as(user)
       visit dashboard_path
 
-      click_on 'All Ready To Pair'
+      click_on 'Ready To Pair'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.